framework/build.gradle

```diff
@@ -25,12 +25,24 @@ dependencies {
     implementation project(':dataflow')
     implementation project(':javacutil')
     implementation files("${stubparserJar}")
-    implementation files("${afuJar}")
+    // AFU is an "includedBuild" imported in checker-framework/settings.gradle, so the version number doesn't matter.
+    // https://docs.gradle.org/current/userguide/composite_builds.html#settings_defined_composite
+    implementation('org.checkerframework:annotation-file-utilities:*') {
+        exclude group: 'com.google', module: 'javac'
+    }
     implementation project(':checker-qual')
-    // As of 2019/12/16, the version of reflection-util in the Annotation
-    // File Utilities takes priority over this version, in the fat jar
-    // file. :-( So update it and re-build it locally when updating this.
     implementation 'org.plumelib:reflection-util:0.2.2'
+    implementation 'org.plumelib:plume-util:1.1.4'
+
+    // TODO: The annotated version of Guava gives an error:
+    // org.checkerframework.annotatedlib/guava/.../guava-28.2-jre.jar(/com/google/common/collect/ImmutableCollection.class):
+    // warning: Cannot find annotation method 'value()' in type 'DoNotMock': class file for com.google.errorprone.annotations.DoNotMock not found
```

Comment:
This seems to be a bug in how the annotated version of Guava was made.  I don't understand it yet.
In the meanwhile, can you try the [workaround used in AFU](https://github.com/typetools/annotation-tools/commit/cad4722172f7fc93436d8af0ba9a6eb9b1ba6cd0), which is to add a dependency on `error_prone_annotations`?